manager/controllers/app/plotter_generator.go

```diff
@@ -142,17 +142,21 @@ func (p *PlotterGenerator) AddFlowInfoForAsset(item DataInfo, appContext *app.Fy
 
 	// DataStore for destination will be determined if an implicit copy is required
 	var sinkDataStore *app.DataStore
+	dataitem := item.Context.CatalogService + "/" + item.Context.DataSetID
```

Comment:
CatalogService is an optional field. Since we don't have multiple catalog services in the same deployment, we don't use this field at all. Printing DataSetID is enough.

Reply:
If we store the logs and use them after the deployment no longer exists, this will enable us to know from which data catalog the dataset came.

Reply:
Then populate this field from https://github.com/shlomitk1/the-mesh-for-data/blob/5a90ce6490c151ab3741f257dfb3e95d66e07dcc/manager/controllers/utils/configuration.go#L27 in case it is empty.

Reply:
@shlomitk1 Why do we have it in the item struct if it's not being populated?

Reply:
> @shlomitk1 Why do we have it in the item struct if it's not being populated?

As the first step to support multi-catalog setup. At least once we had an issue to support it. The idea was that if an asset is registered in another catalog (not the one in the setup), the catalog service will be populated. An empty field indicates that the asset is registered in the "official" catalog.
I am not aware of a plan to support more than one catalog, so maybe this field is not required. @roytman @roee88 - thoughts?